test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/DetectRedundantSuppressionsInMembersAndTypes.cs

```diff
@@ -0,0 +1,179 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq.Expressions;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+
+namespace Mono.Linker.Tests.Cases.Warnings.WarningSuppression
+{
+	[ExpectedNoWarnings]
+	[SkipKeptItemsValidation]
+	public class DetectRedundantSuppressionsInMembersAndTypes
+	{
+		public static void Main ()
+		{
+			RedundantSuppressionOnType.Test ();
+			RedundantSuppressionOnMethod.Test ();
+			RedundantSuppressionOnLocalMethod.Test ();
+			RedundantSuppressionOnNestedType.Test ();
+			RedundantSuppressionOnProperty.Test ();
+			MultipleRedundantSuppressions.Test ();
+			RedundantAndUsedSuppressions.Test ();
+			DoNotReportNonLinkerSuppressions.Test ();
+			DoNotReportSuppressionsOnMethodsConvertedToThrow.Test ();
+			SuppressRedundantSuppressionWarning.Test ();
+		}
+
+		public static Type TriggerUnrecognizedPattern ()
+		{
+			return typeof (DetectRedundantSuppressionsInMembersAndTypes);
+		}
+
+		public static string TrimmerCompatibleMethod ()
+		{
+			return "test";
+		}
+
+		[ExpectedWarning ("IL2121", "IL2071", ProducedBy = ProducedBy.Trimmer)]
+		[UnconditionalSuppressMessage ("Test", "IL2071")]
+		public class RedundantSuppressionOnType
+		{
+			public static void Test ()
+			{
+				DetectRedundantSuppressionsInMembersAndTypes.TrimmerCompatibleMethod ();
```

Comment:
```suggestion
				TrimmerCompatibleMethod ();
```
There should no need to have the class name here

Reply:
Simplified method calls across the tests.